ginga/web/pgw/ImageViewPg.py

```diff
@@ -91,8 +91,8 @@ def choose_best_renderer(self):
 
         raise ImageViewPgError("No valid renderers available: {}".format(str(self.possible_renderers)))
 
-    def update_image(self):
-        self.logger.debug("update_image pgcanvas=%s" % self.pgcanvas)
+    def update_widget(self):
```

Comment:
So, this is renamed without deprecation. Will this break anything downstream?

Reply:
Good catch. It's an internal routine and unlikely to be called by user code, but I'll add a compatibility line in the base class.

Reply:
Hmm.. I already did add that! See line 2868 of the updated `ImageView.py`.

Reply:
flake8 caught me adding a duplicate just now...

Reply:
Oh, ops. Sorry for the noise!

Reply:
No worries, that's what reviews are for! I don't think the user API changed--at least, I kept it in my mind that should be so.  This PR is mostly just internal rendering refactoring.